pkg/taskrunmetrics/metrics.go

```diff
@@ -473,6 +474,17 @@ func (r *Recorder) CloudEvents(ctx context.Context, tr *v1beta1.TaskRun) error {
 	return nil
 }
 
+func isPartOfPipeline(tr *v1beta1.TaskRun) (bool, string, string) {
```

Comment:
could you add a docstring stating that this returns the values of the pipeline label and pipelinerun label if it is part of a pipeline?